pom.xml

```diff
@@ -52,6 +52,11 @@
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
         </dependency>
+        <dependency>
```

Comment:
spróbujmy dependency management dla Testcontainers i być może innych bibliotek

Reply:
Wydaje mi się, że do TC samodzielnie spring parent to ogarniał. Dodawałem boma, ale wtedy rozpierdzielały się dwie dependency (ta od testcontainers junit i jeszcze jakaś jedna)

Reply:
OK, niech zostanie tak żeby działało dobrze 👍